src/conn.c

```diff
@@ -256,7 +256,7 @@ int conn_get_addrs(juice_agent_t *agent, addr_record_t *records, size_t size) {
 	return get_mode_entry(agent)->get_addrs_func(agent, records, size);
 }
 
-static int juice_mux_stop_listen(const char *bind_address, int local_port) {
```

Comment:
Did this need to be `static`?  `juice_mux_listen` is not `static` so I removed it from here otherwise it doesn't compile.